tests/platform_tests/mellanox/mellanox_thermal_control_test_helper.py

```diff
@@ -260,6 +268,17 @@ def deinit(self):
                 logging.error(error_message)
                 raise RuntimeError(error_message)
 
+    def is_201911(self):
```

Comment:
what do you think if we check the kernel version of the image? 4.9 -> 201911 4.19-->master ?

Reply:
Yes, actually to check the kernel version is a better solution for this case. However, I feel that we might still need this parameter "--sonic_branch_name"" in future. We will continue have release branch and master branch in future, and there could be features that work different on different branch. As long as we don't have separate sonic-mgmt branch for different sonic-buildimage branch, we might still need this parameter. Any suggestion?